scripts/python-skywater-pdk/skywater_pdk/subscript_to_math.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python3
```

Comment:
I would not say this script is *actually* related to the SkyWater PDK (it just happens to be used here)? Probably belongs in some tooling around conversion to the rst format?

Reply:
Maybe it would make sense just to commit the patched rst files with converted equations?

Reply:
I've removed the script and updated the equations to include units.
eg. `:math:`I_E=10 µA``

Should 'A to B' ranged be included as well?
current:  
`:math:`|V_{c0} – V_{c1}| = 0` to 5.0V`
possible option:
`:math:`|V_{c0} – V_{c1}| = 0 to 5.0V``

Reply:
@mithro PTAL